### PR TITLE
Add PyUnfold

### PR DESCRIPTION
**Pull request by jrbourbeau**
Add PyUnfold package to conda-forge

### CODE REVIEW

**Comment by synapticarbors** [comment]
Package seems to be broken. When I look at the source distribution, the file is `Unfold.py` rather than `unfold.py` that looks to be expected from the imports.  Same issue with `Utils.py` vs `util.py`. Not sure what the origin of this is.

File: recipes/pyunfold/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/pyunfold/meta.yaml]: Consider using `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Comment (jrbourbeau) [reply recipes/pyunfold/meta.yaml]: Will do

**Comment by jrbourbeau** [comment]
Thanks for the review @synapticarbors!

> When I look at the source distribution, the file is Unfold.py rather than unfold.py that looks to be expected from the imports. Same issue with Utils.py vs util.py. Not sure what the origin of this is.

That was an oversight on my part and has been fixed. Thank you for pointing that out.

**Comment by jrbourbeau** [comment]
Just checking in. Is there anything else that should be done on my end?

**Comment by synapticarbors** [comment]
@jrbourbeau -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**